.travis.yml

```diff
@@ -7,9 +7,5 @@ node_js:
   - 8
   - 10
 
-cache:
-  directories:
-    - node_modules
-
```

Comment:
i guess caching is being nice to travis, and speeds up our builds,
why not be nice and fast ?

Reply:
I agree, but this is more a `package-lock.json` problem. npm can be a bit finicky with installing dependencies and using the `package-lock.json` file. Almost a year after npm introduced the `package-lock.json` file it's still a very common thing to do when having dependency issues, to `rm -rf node_modules package-lock.json`.

Speed doesn't really seem to be an issue anyway, this package is fairly small and the build for this PR took just over 1 minute and if we look at the builds from `master` they take about 1 and a half minute.

Hence removing the cache just ensures all the dependencies are installed cleanly and apparently it's faster.

Reply:
following that logic shouldn't we remove package-lock.json too ?